src/components/TodoList.js

```diff
@@ -3,17 +3,15 @@ import TodoForm from './TodoForm';
 import Todo from './Todo';
 
 function TodoList() {
-    const [todos, setTodos] = useState([]);
```

Comment:
changing the default value from empty array `[]` to `{}` will cause error, as with line:13 we are trying to insert the array in a setter which is expecting an object